resources/faq.md

```diff
@@ -78,15 +78,11 @@ For more accurate features and issues, everything is detailed in the issues of a
 
 ## I have just updated Meilisearch, and I am getting an error: "Cannot open database, expected Meilisearch engine version..."
 
-Until our first stable release (v1.0), Meilisearch minor versions are not compatible with each other. This means **every new version is considered breaking** with the small exception of bug-fixing patches. To fix this error, simply delete your database folder (`data.ms` by default) and re-index your documents with the current-version engine. See [updating Meilisearch](/learn/advanced/updating.md) for more information.
+Meilisearch major versions introduce breaking changes and are not compatible with each other. Meilisearch version 0 releases, such as v0.30 or v0.24, are also not compatible with each other. Consult the [updating guide](/learn/advanced/updating.md) for more information on how to safely migrate your data between Meilisearch releases.
 
 ## What are the recommended requirements for hosting a Meilisearch instance?
 
-**The short answer: we don't know yet!**
-
-Indexing documents is a highly complex process, making it difficult to accurately estimate the size and memory use of a Meilisearch database. At this stage, our engine is changing constantly, which means benchmarks are constantly out of date.
-
-Once our engine is completely stable (v1), we will provide detailed benchmarks with different datasets. Until then, read on to learn how to optimize your Meilisearch instance.
+Indexing documents is a complex process, making it difficult to accurately estimate the size and memory use of a Meilisearch database. There are a few aspects to keep in mind when optimizing your instance.
```

Comment:
Would suggest we copy some text from the Crisp helpdesk article (which itself copies a lot of text from this answer). Here's my adaptation:

```suggestion
**The short answer:**

The recommended requirements for hosting a Meilisearch instance will depend on many factors, such as the number of documents, the size of those documents, the number of filters/sorts you will need, and more. For a quick estimate to start with, try to use a machine that has at least ten times the disk space of your dataset.

**The long answer:**

Indexing documents is a complex process, making it difficult to accurately estimate the size and memory use of a Meilisearch database. There are a few aspects to keep in mind when optimizing your instance.
```

Ideally these would be headers, not simply bolded text. The only reason I didn't suggest it that way is because it would probably require changing the following H3s to H4s.

Reply:
I think the short answer is actually all we could reasonably need here.

Given v1 is just around the corner, I'll accept @dichotommy's suggestion. Then, at a later date, I'll move unique info to a better location and delete the rest.